packages/discord.js/typings/index.d.ts

```diff
@@ -5897,27 +5849,27 @@ export interface GuildScheduledEventCreateOptions {
 }
 
 export type GuildScheduledEventRecurrenceRuleOptions =
-  | BaseGuildScheduledEventRecurrenceRuleOptions<
+  | _BaseGuildScheduledEventRecurrenceRuleOptions<
       GuildScheduledEventRecurrenceRuleFrequency.Yearly,
       {
         byMonth: readonly GuildScheduledEventRecurrenceRuleMonth[];
         byMonthDay: readonly number[];
       }
     >
-  | BaseGuildScheduledEventRecurrenceRuleOptions<
+  | _BaseGuildScheduledEventRecurrenceRuleOptions<
       GuildScheduledEventRecurrenceRuleFrequency.Monthly,
       {
         byNWeekday: readonly GuildScheduledEventRecurrenceRuleNWeekday[];
       }
     >
-  | BaseGuildScheduledEventRecurrenceRuleOptions<
+  | _BaseGuildScheduledEventRecurrenceRuleOptions<
       GuildScheduledEventRecurrenceRuleFrequency.Weekly | GuildScheduledEventRecurrenceRuleFrequency.Daily,
       {
         byWeekday: readonly GuildScheduledEventRecurrenceRuleWeekday[];
       }
     >;
 
-type BaseGuildScheduledEventRecurrenceRuleOptions<
+export type _BaseGuildScheduledEventRecurrenceRuleOptions<
```

Comment:
why the underscore prefix, considering that's an actual type used, exported and documented?